bin/unit_test

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+for arg in "$@"
+do
+    if [ "$arg" == "--help" ] || [ "$arg" == "-h" ]
+    then
+        echo "Test options: all, typescript, kotlin, swift, python"
+        echo "Multiple languages can be specified at the same time"
+        echo "'all' is the default"
+    fi
+done
+
+if [ $# == 0 ] || [ "$1" == "all" ]
+then
+  tests="typescript kotlin swift python"
```

Comment:
It would be nice to have an entry point that unittests the `packages/sdk-codegen*` packages as well. Maybe add a `"codegen"` argument here if can we combine the tests/results or `"codegen codegen-utils codegen-scripts"` arguments if we can't?

Reply:
Agreed. That would be another PR.